pkg/challenges/challenges_test.go

```diff
@@ -66,6 +72,111 @@ func Test_isSpiffeIDAllowed(t *testing.T) {
 	}
 }
 
+func TestUri(t *testing.T) {
+	cfg := &config.FulcioConfig{
+		OIDCIssuers: map[string]config.OIDCIssuer{
+			"https://accounts.domain.com": {
+				IssuerURL:     "https://accounts.domain.com",
```

Comment:
I think this and other uses of `domain.com` should be `example.com` (or `example.net` when testing for mismatches with `example.com`).

Reply:
Changed